exercises/concept/ellens-alien-game/.docs/instructions.md

```diff
@@ -0,0 +1,94 @@
+# Instructions
+
+Ellen is making a game where the player has to fight aliens.
+She has just learned about Object Oriented Programming (OOP) and is eager to take advantage of what using `classes` could offer her program.
+
+To Ellen's delight, you have offered to help and she has given you the task of programming the aliens that the player has to fight.
+
+
+## 1. Create the Alien Class
+
+Define the Alien class with a constructor that accepts two `int` parameters `x` and `<y>`, putting them into `x_coordinate` and `y_coordinate` member variables.
+Every alien will also start off with a health level of 3, so the `health` variable should be initialized as well.
+
+`health` should be a private variable.
+To let other parts of the program read the health information, Ellen wants to have a `public` `get_health()` method which returns an `int`.
+
+```cpp
+Alien alien{2, 0};
+alien.x_coordinate;
+// => 2
+alien.y_coordinate;
+// => 0
+alien.get_health();
+// => 3
+```
+
+Now, each alien should be able to internally track its own position and health.
+
+## 2. The `hit` Method
+
+Ellen would like the Alien `class` to have a `hit` method that decrements the health of an alien object by 1 when called.
+This way, she can simply call `<alien>.hit()` instead of having to manually change an alien's health.
```

Comment:
I'm not sure about the angle brackets here.  
I've seen students who think they are part of the syntax.
How about something like `some_alien_instance` instead?